src/commands/context/list.ts

```diff
@@ -82,11 +75,9 @@ export async function list(
 
   spinner.stop()
 
-  const endTime = new Date().getTime()
+  endTime = new Date().getTime()
```

Comment:
`endTime` has been assigned a value only once, that is why it should be a `const`

Reply:
fixed